operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -31,6 +31,8 @@ Entries should include a reference to the Pull Request that introduced the chang
 
 * [CHANGE] Memcached: Update to Memcached 1.6.34. #10318
 * [ENHANCEMENT] Minio: update subchart to v5.4.0. #10346
+* [ENHANCEMENT] Alertmanager, compactor, ingester and store-gateway statefulsets can configure their PVC template name via the corresponding *.persistentVolume.name values. #10376
+* [ENHANCEMENT] Added option to enable StatefulSetAutoDeletePVC for StatefulSets for compactor, ingester, store-gateway,
```

Comment:
`for compactor, ingester, and store-gateway.` ?

Reply:
This was a copy&paste from an old entry in the changelog. Now, 24 hours latter, I'm not sure why I did that. It's now gone, thank you for noticing 🙌